src/main/java/sqltoregex/ConverterManagement.java

```diff
@@ -0,0 +1,123 @@
+package sqltoregex;
+
+import net.sf.jsqlparser.JSQLParserException;
+import net.sf.jsqlparser.parser.CCJSqlParserUtil;
+import net.sf.jsqlparser.statement.Statement;
+import net.sf.jsqlparser.util.deparser.ExpressionDeParser;
+import net.sf.jsqlparser.util.deparser.StatementDeParser;
+import net.sf.jsqlparser.util.validation.Validation;
+import net.sf.jsqlparser.util.validation.ValidationError;
+import net.sf.jsqlparser.util.validation.feature.DatabaseType;
+import org.springframework.stereotype.Service;
+import sqltoregex.deparser.StatementDeparser;
+
+import java.util.*;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+
+/**
+ * realize a spring service for handling the converting process
+ */
+@Service
+public class ConverterManagement {
+    /**
+     * Validate inserted SQL-statements for oracle, mysql, sqlserver, mariadb
+     * return true or false, with a console output about the error messages
+     * @param sqlstatement String
+     * @return Boolean
+     */
+    public Boolean validate(String sqlstatement){
+        Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+        logger.setLevel(Level.ALL);
+        Handler handler = new ConsoleHandler();
+        handler.setLevel(Level.ALL);
+        logger.addHandler(handler);
+
+        List<DatabaseType> supportedDBMS = new ArrayList<>();
+        supportedDBMS.add(DatabaseType.ORACLE);
+        supportedDBMS.add(DatabaseType.MYSQL);
+        supportedDBMS.add(DatabaseType.SQLSERVER);
+        supportedDBMS.add(DatabaseType.MARIADB);
+
+        Validation validation = new Validation(supportedDBMS, sqlstatement);
+        List<ValidationError> validationerrors = validation.validate();
+        if(validationerrors.isEmpty()) { return true;}
+        else{
+            for(ValidationError va : validationerrors){
+                logger.log(Level.WARNING, () -> "Something went wrong: " + va.toString());
+            }
+            return false;
+        }
+    }
+
+    private Statement parse(String sqlstatement) throws JSQLParserException {
+        return CCJSqlParserUtil.parse(sqlstatement);
+    }
+
+    /**
+     * compared list elements about unique elements
+     * @param list List<String>
+     * @return boolean
+     */
+    private Boolean checkOfDistinctElements(List<String> list){
+        if(list.isEmpty()) throw new IllegalArgumentException("List should not be empty. One element is required.");
+        Set<String> set = new HashSet<>(list);
+        return (set.size() == list.size());
```

Comment:
Problem might be, that List contains 1 duplicate and 2 same Strings.
Then set.size() is 1 less then list.size() but still has duplicates.